api/src/services/users/users.js

```diff
@@ -1,4 +1,5 @@
 import { db } from 'src/lib/db'
+import { sendEmail } from 'src/lib/email'
```

Comment:
This is failing the lint step in CI because it's not being used; did you have plans to use it in this file?

```suggestion
```

Reply:
I did and then decided against it... Thanks for catching it